R/lagwalk.R

```diff
@@ -23,16 +23,26 @@ train_lagwalk <- function(.data, specials, ...) {
   }
 
   fitted <- c(rep(NA, min(lag, n)), utils::head(fits, -lag))
-  if (drift) {
-    fit <- summary(stats::lm(y - fitted ~ 1, na.action = stats::na.exclude))
-    b <- fit$coefficients[1, 1]
-    b.se <- fit$coefficients[1, 2]
-    sigma <- fit$sigma
+
+  # In general drift is a logical but could also be a numeric if RW model with drift should not be re-estimated. 
```

Comment:
I think the drift input should be type stable. The typical interface for specifying coefficient values is using the `fixed` parameter. For lagwalk models, I would expect it to be `RW(y ~ drift(fixed = b)`.

I can make these changes if you'd like.

Reply:
So meaning you want to have a further argument called fixed in the function `drift()`, which is defined in the specials in `RW `and `SNAIVE`? 

If so I would suggest that the return of the function drift is a list where the first list entry is names _drift_ with TRUE or FALSE (as it is) and the second entry is named **fixed** which is NULL by default.

What do you think?

Reply:
That is correct, so the structure would be `drift(drift = TRUE, fixed = FALSE)`, which returns a list of parameters that can be used by `train_lagwalk()` and others as needed.